src/config/configuration.ts

```diff
@@ -46,11 +47,11 @@ const configuration = () => {
     modulePath: "./loggingProviders/defaultLogger",
     config: {},
   };
-  const jsonConfigMap: { [key: string]: object | object[] | boolean } = {
+  const jsonConfigMap: { [key: string]: unknown } = {
```

Comment:
Why moving to a less strict definition?

Reply:
YAML `load` has return type `unknown`, as opposed to `any` with JSON.parse. This requires tighter type checking. I could have done a cast here and kept the previous signature, but keeping it unknown here seemed more accurate. It gets cast anyways in `configService.get`, and non-object json/yaml files will throw runtime errors as before.